### ISSUE TITLE
how can i define unsigned column?

### ISSUE THREAD

**Issue by sajjad-shirazy** [typeorm/typeorm#438]

**Comment by pleerock** [comment]
custom column types are not supported yet. But its planned in the future

**Comment by pleerock** [comment]
All database specific column types are supported now since `0.1.0-alpha.10`

**Comment by jdpalumbo** [comment]
@pleerock How exactly is this achieved? I can not seem to find any documentation or property in the `ColumnOptions` interface for defining an unsigned column?

**Comment by godvsdeity** [comment]
@pleerock i'm wondering the same, how do i get typeorm to generate the following migration code `id INT UNSIGNED NOT NULL AUTO_INCREMENT`

**Comment by vkosachev** [comment]
Are there any workarounds on this ?

**Comment by AlexMesser** [comment]
`UNSIGNED` and `ZEROFILL` attributes avaliable from `typeorm@0.2.0-alpha.40`. You can try it via `npm i typeorm@next`. Usage:

```typescript
@Entity()
export class Post {

    @PrimaryColumn({ unsigned: true })
    id: number;

    @Column({ zerofill: true })
    num: number;

}
```

**Comment by d-yokoi** [comment]
@AlexMesser 
I want to add `UNSIGNED` attribute on `PrimaryGeneratedColumn` decorated property but it's not supported right?

**Comment by AlexMesser** [comment]
fixed and released in `typeorm@0.2.0-alpha.45`. You can try it via `npm i typeorm@next`.

**Comment by d-yokoi** [comment]
Thank you! I'll use it.
